perf/istio/values.yaml

```diff
@@ -10,16 +10,18 @@ global:
   imagePullPolicy: Always
   k8sIngressSelector: ingressgateway
 
+  controlPlaneSecurityEnabled: true
```

Comment:
In separate PR - check the pods, one of them overrides the default to enable control plane security. We should do the opposite - disable ( so we cover both enabled and disabled - as will be the case when people transition )